net-libs/shairplay/shairplay-0_pre20160304.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="6"
+
+inherit autotools 
+
+if [[ ${PV} == "9999" ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="git://github.com/juhovh/shairplay.git"
+else
+	EGIT_COMMIT="498bc5bcdd305e04721f94a04b9f26a7da72673f"
+	SRC_URI="https://github.com/juhovh/shairplay/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz"
+	S="${WORKDIR}/${PN}-${EGIT_COMMIT}"
```

Comment:
...I think it's fine as it is for now. I tested it against media-tv/kodi with and without openssl installed - works in both cases.